runtime/runtime-params-estimator/src/gas_cost.rs

```diff
@@ -23,12 +22,9 @@ pub(crate) struct GasCost {
     /// The smallest thing we are measuring is one wasm instruction, and it
     /// takes about a nanosecond, so we do need to account for fractional
     /// nanoseconds here!
-    time_ns: Ratio<u64>,
+    time_ns: Option<Ratio<u64>>,
     // Values used for `GasMetric::ICount`
-    instructions: Ratio<u64>,
-    io_r_bytes: Ratio<u64>,
-    io_w_bytes: Ratio<u64>,
-    metric: GasMetric,
+    qemu_measurement: Option<QemuMeasurement>,
```

Comment:
Maybe just `qemu`? The call-sites look pretty repepetitive

Reply:
Good idea, it mildly improves some of the verbose code around this.